src/test/java/org/opentripplanner/middleware/tripMonitor/jobs/CheckMonitoredTripTest.java

```diff
@@ -80,23 +80,16 @@ public static void tearDown() {
     }
 
     /**
-     * To run this trip, change the env.yml config values for OTP_SERVER
+     * To run this trip, change the env.yml config values for OTP_API_ROOT
      * (and OTP_PLAN_ENDPOINT) to a valid OTP server.
      */
     @Test
     public void canMonitorTrip() {
         // Do not run this test on Travis CI because it requires a live OTP server
         // FIXME: Add live otp server to e2e tests.
         assumeTrue(!isRunningCi && isEndToEnd);
-        // Submit a query to the OTP server.
-        // From P&R to Downtown Orlando
-        OtpDispatcherResponse otpDispatcherResponse = OtpDispatcher.sendOtpPlanRequest(
-            "28.45119,-81.36818",
-            "28.54834,-81.37745"
-        );
-        // Construct a monitored trip from it.
-        MonitoredTrip monitoredTrip = new MonitoredTrip(otpDispatcherResponse)
-            .updateAllDaysOfWeek(true);
+        MonitoredTrip monitoredTrip = TestUtils.constructedMonitoredTripFromOtpResponse();
```

Comment:
This method name does not seem right. Also, I find the combination of the OTP plan request and construction of MonitoredTrip into this method not entirely useful. To me it just obfuscates what's happening without really providing much benefit. I think a cleaner refactor would be to encapsulate the test plan request with sample coordinates into its own method (e.g., `TestUtils.sendSamplePlanRequest`) and then just create the monitored trip from the response directly in this test method's body (as was previously done).

Reply:
Updated.